Gemfile.lock

```diff
@@ -258,7 +258,7 @@ GEM
       signet (~> 0.8)
       typhoeus
     builder (3.2.4)
-    bulkrax (5.3.0)
+    bulkrax (5.4.1)
```

Comment:
the changes on this version worked for myself and others who attended session 5 of the samvera training workshop. however, both rob and jeremy believe it will actually break something. instead, some env variable or something may need to be set to rectify the issue I saw in `v5.4.0`.

blocking this from being merged until the 2 issues below are resolved and a new release is cut:
- https://github.com/samvera-labs/bulkrax/issues/866
- https://github.com/samvera-labs/bulkrax/issues/867